routes/UserSignIn.js

```diff
@@ -0,0 +1,10 @@
+var express = require('express');
+var router = express.Router();
+var path = require('path');
+
+/* GET home page. */
+router.get('/', function(req, res) {
+    res.sendFile (path.join('C:\\Users\\bisho\\Desktop\\LUCCA\\views\\UserSignIn.html'))
```

Comment:
Also the path to the file is absolute, and looks like it will only work on your machine. You will need to make this a relative path.